setup.py

```diff
@@ -11,6 +11,7 @@
 setup(
     name='buza-website',
     description='buza',
+    version='0.0.1',
```

Comment:
We should rather keep `use_scm_version`, but PR #78 should address this. :coffee: